### DISCUSSION TITLE
広範な接触をお知らせする価値：　行動変容の実例から

### DISCUSSION THREAD

**Discussion by kvaluation** [cocoa-mhlw/cocoa#859] [Ideas]
接触日シート別冊を運営している @kvaluation 鈴木健治です。

広範な接触（1m以内、15分以上の条件を満たさない陽性登録者さんとのすれ違い）があった方へのアンケートを開始しました。

■結果スクショ
<img width="893" alt="広範な接触アンケ01" src="https://user-images.githubusercontent.com/60703087/152728259-71fca4aa-0ebb-4fda-8232-d477ef567351.png">

■このGoogleフォームで入力してもらっています。
https://docs.google.com/forms/d/e/1FAIpQLSfBa28En6GdukQG7yJTTO6UtmntKU-zAVs3DawCs7DLInwE2A/viewform?usp=sf_link

■Google スプレッドシート経由で、データポータルに自動集計・自動更新（15分間隔）しています。
https://datastudio.google.com/reporting/069598a2-3f01-4b51-b023-cdb478992182/page/p_uuo7y60xrc

★ご報告
COCOAで接触通知がでなくとも、陽性者さんとのすれ違いが発生していますという情報をお伝えする（注１）ことで、行動記録を始めたり、買い物の頻度を落としたり、人との距離に気をつけたり、などのコロナ社会への適応をしてくださっている方々がいらっしゃいます。

COOCA2にて、動作報告としてのお知らせであっても、波及的効果として、コロナ社会の波を「乗りこなす人々（注２）」に正確な情報をお伝えする価値があることが、アンケート結果からも、示唆されております。

デメリットの最小化を工夫しつつ、COCOA2にて広範な接触に関する情報、特に接触日ごとのすれ違いの有無がわかる情報が提供なされれば、このような行動変容・注意心の向上や最適化（予防に必要な注意を払うよう努めること）をうながせるものと考えます（注３）。

○注１　COCOAログチェッカーの存在により、広範な接触の存在が広まりました。河原さんに感謝しています。
○注２　山口 一郎氏（サカナクション）「アダプトツアー　武道館　オンライン配信」でのコメントより。波に流されるのではなく乗りこなすというメッセージに励まされて、鈴木健治としてはしなくても問題無いこのアンケートを、してみることとしました。
○注３　厚生労働省設置法第4条第1項第19号, 3号、感染症の予防及び感染症の患者に対する医療に関する法律（感染症法）第15条, 16条等。第4条。

**Comment by parasite2006** [comment]
私はCOCOAの陽性登録が正式再開された2020年7月15日以降、陽性登録件数データ更新の都度新規陽性確認者数に対する割合を陽性登録率として算出しグラフ化してきましたが、この値は2022年4月7日のCOCOA v2.0.0の公開以来（大型連休中の検査停止による急降下と連休明けの反跳を除いて）基本的に低下の一途を辿っています。2021年12月に陽性登録用処理番号の自動発行SMS通知が開始されて以降は開始前と比べて1桁上昇し、一時は19.4%まで上がっていたのに、5/27（金）の値は13.1%です。
<img width="1174" alt="COCOA%lognew(20220527)" src="https://user-images.githubusercontent.com/84975567/171602465-dcc2655d-4132-4fe2-80bd-870676a1a00d.png">
陽性登録1件あたりの提供接触キー数は陽性登録率が低い時は高く、陽性登録率が上がると低下する傾向がありますが、これもCOCOA v2.0.0の公開以降上がり続けています。
<img width="688" alt="TEKs:PositiveRegistrations(20220528)" src="https://user-images.githubusercontent.com/84975567/171603660-96071839-e43d-44db-a08e-4dd5889191a4.png">
（書き込みの「4月4日」は「4月7日」のミスプリントです）

COCOA v2.0.0のiOS版で接触ログから本体表示基準未満の陽性登録者との接触の有無を知ることができなくなっていることを知ってCOCOAを削除したというiPhoneユーザーを、Twitter上ですでに複数見かけています。「接触確認アプリ本体に接触通知が表示されたら無料検査」という2020年8月21日付厚労省事務連絡
https://www.mhlw.go.jp/content/000661724.pdf
が地元の流行状況と検査能力の兼ね合いで空文化されて以来、このアプリを使用する唯一のメリットは表示基準未満の接触の有無を接触ログから知れることだったのに、iOS版からこの機能がなくなったことはiPhoneユーザーにとってアプリ使用のメリットが消え失せたことを意味します。この機能を一刻も早く復活しない限り、日本国内のスマホユーザーの多数を占めるiPhoneユーザーがCOCOAを削除する結果、実ユーザーの減少とこれによる陽性登録率の低下に歯止めがかからなくなるでしょう。

**Reply by keiji** [reply]
データありがとうございます。

まず結論からお伝えすると、 #605 「閾値未満で接触通知の対象にならなかった接触を確認出来る画面を作る」については行政内で廃案となり、COCOAには実装しないという結論となったという連絡を今週の初めに受けました。この決定により現在COCOAに実装され、無効化されている「閾値未満の接触（広範な接触）の表示」に関するコードは、すべて取り除かれる予定であると聞いています。

閾値未満の接触について表示する意義については、個人的には十二分に理解しているつもりです。
もともとENv1を使っていたCOCOA1系では接触を検出した時点でシステムからの通知が表示されるために実現できなかったこの機能ですが、ENv2に移行することにより、通知の表示がCOCOAからだけになることで実現可能となりました。

閾値未満の接触が表示できれば、通知がなくてもCOCOAを起動する利用者が増え（アクティブユーザーの増加と接触に対する意識の高まり）、「本当に接触を検出するのか」とか「身の回りに陽性者が出たのに接触通知が出なかった」という利用者に一定の説明（閾値未満の接触であったこと）をすることで、不信を解消することが期待できる。

個人的には、COCOA v2.0.0の目玉機能と考え、野上さん、林さんと話し合いながら実装をしました。クローズドβの形で提供し、表示方法についてはDiscussionを通じてコントリビューターの皆さんの意見を頂きながら改善を重ねました。しかしながら、最終的に利用者に届けられないという結論となったことは忸怩たる思いです。

その上で、閾値以上・未満に依らず、検出した接触についてはCOCOAの中にデータとして保存しています。
「接触通知アプリ」という透明性が求められるアプリにおいて、アプリが保存しているデータにアクセスする権利は利用者に認められるべきであると個人的に考えています。この観点では、行政内でも異論はないものと認識しています。

次バージョン（v2.0.1）では、接触データの保存機能を提供することが予定されています。保存した接触データを見ることで、閾値未満の接触についても知ることができます。

 - https://github.com/cocoa-mhlw/cocoa/pull/931

保存されるデータはJSONで、見方については画面に表示するより複雑になりますが、工夫の余地はあると考えています。
COCOAログチェッカーのような外部ツールを発展させるような方法が1つの選択肢としてあり得るのではないかとも考えています。COCOAで表示するという、もっともシンプルな選択肢がなくなったことは痛恨の極みではありますが、残された可能性について検討を進めていきます。

**Reply by parasite2006** [reply]
詳細なご説明ありがとうございました。

旧版COCOAのように陽性登録者との接触回数をアプリ入りスマホ上でチェックすることができなくなると確定したのは残念なことではありますが、#931 の見本画面のようにアプリ本体上の接触確認画面上でボタンを押すと（このボタンは本体表示基準を超える接触が確認できなかった場合にこそ使える状態になっていてほしい）接触データがJSONファイルに書き出されて決まった場所に保存されるようになってさえいれば、このファイルをCOCOAログチェッカーのようなツールで解析できるようにすることは可能なので、これまで旧版COCOAとCOCOAログチェッカーを併用してきたiPhoneユーザーに納得して新版COCOAを使ってもらえることだろうと期待します。
是非早期実現をよろしくお願いいたします。
